cicd/app/rasd_static.yaml

```diff
@@ -77,16 +77,16 @@ Resources:
         HostedZoneId: Z2FDTNDATAQYW2
       HostedZoneName: !Sub "${pHostedZone}."
 
-  RootRasdStaticDnsRecord:
```

Comment:
temporarily commented out the creation of the root domain and it's being used in testing for the service.